about.html

```diff
@@ -1,291 +1,71 @@
-<html>
-<head>
-	<title>MCEdit Unified</title>
-	<script src="js/jquery.min.js"></script>
-	<script src="js/bootstrap.min.js"></script>
-	<script src="js/scripts.js"></script>
-	<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
-	<link rel="stylesheet" href="css/font-awesome.min.css">
-	<link rel="stylesheet" href="css/stylesheet.css">
-
-	<script>
-	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-	  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-	  ga('create', 'UA-57749723-1', 'auto');
-	  ga('send', 'pageview');
-
-	</script>
-
-</head>
-<body>
-	<div class="container">
-		<div align="center">
-			<img src="doc/images/favicon.png">
+<!DOCTYPE html>
+<html lang="en-US">
+	<head>
+		<!--[if lte IE 9]>
+		<p class="update">
+			You are using an out-of-date browser, please update or change your current browser to <a href="https://www.google.com/chrome/browser/desktop/index.html">Google Chrome</a> (Window XP users are supported),
+			<a href="https://www.mozilla.org/en-US/firefox/new">FireFox</a> (Window XP users are supported) or <a href="http://windows.microsoft.com/en-us/internet-explorer/download-ie">Internet Explorer (IE)</a> 11 for systems using Windows 7,8,8.1. 
+			<br> Check out <a href="http://browsehappy.com/">Browsehappy.com</a> to see the latest version of your browser.
+		</p>
+		<![endif]-->
+		<meta charset="utf-8">
+		<title>MCEdit Unified - About</title>
+		<script src="js/jquery.min.js"></script>
+		<script src="js/bootstrap.min.js"></script>
+		<script src="js/scripts.js"></script>
+		<link rel="stylesheet" href="css/bootstrap.min.css" type="text/css">
+		<link rel="stylesheet" href="css/font-awesome.min.css">
+		<link rel="stylesheet" href="css/stylesheet.css">
+		<link rel="stylesheet" href="css/text.css">
+		<link rel="shortcut icon" href="favicon.ico">
+		<script>
+		  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+		  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+		  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+		  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+		  ga('create', 'UA-57749723-1', 'auto');
+		  ga('send', 'pageview');
+		</script>
+	</head>
+
+	<body>
+		<article>
```

Comment:
I think you should use a `<div class="container">` tag instead of an `<article>` tag, because right now, the `about.html` page looks like this: http://i.imgur.com/tFDuiMd.png